File: todo_back/src/main/java/com/cos/todo/domain/Task.java
package com.cos.todo.domain;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;

import org.hibernate.annotations.ColumnDefault;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Entity
@Data
@Builder
@AllArgsConstructor
@NoArgsConstructor
public class Task {
	@Id
	@GeneratedValue(strategy = GenerationType.IDENTITY)
	private Long id;
	private String title;
	private String content;
	@ColumnDefault("false")
	private boolean completed;
}

File: flutter_todo/lib/domain/task.dart
class Task {
  final int? id;
  final String? title;
  final String? content;
  final bool? completed;

  Task({this.id, this.title, this.content, this.completed});

  Task.fromJson(Map<String, dynamic> json)
      : id = json["id"],
        title = json["title"],
        content = json["content"],
        completed = json["completed"];
}
